fall18/p2/project.py

```diff
@@ -0,0 +1,25 @@
+import csv
+
+
+def getArea(stateName):
+    """ Function that takes in state name and returns the area of the state"""
+    with open('area.csv') as csvDataFile:
+        csvReader = csv.reader(csvDataFile)
+        for row in csvReader:
+            if row[0] == stateName:

```

Comment:
No need to change this, but DictReader is a cleaner way to do this without numerical indexes.